src/pygeoprocessing/geoprocessing.py

```diff
@@ -3703,7 +3665,7 @@ def mask_op(base_array, mask_array):
     os.remove(mask_raster_path)
 
 
-def _gdal_to_numpy_type(band):
+def _gdal_to_numpy_type(gdal_type, metadata):
```

Comment:
Since the function interface is changing, could you update the docstring here as well?